posthog/api/test/test_signup.py

```diff
@@ -642,6 +642,26 @@ def test_api_invite_sign_up_member_joined_email_is_sent_for_next_members(self):
         self.assertEqual(len(mail.outbox), 1)
         self.assertListEqual(mail.outbox[0].to, [initial_user.email])
 
+    @patch("posthog.api.organization.settings.EE_AVAILABLE", False)
+    def test_api_invite_sign_up_member_joined_email_is_not_sent_if_disabled(self):
```

Comment:
I would also add a test to make sure that a) an admin can update the preference in org settings and b) only the admin can update the preference. Let me know if you want me to add those tests.

Reply:
Added